shanoir-uploader/start-shup-linux-mac-java.sh

```diff
@@ -1,12 +1,19 @@
+JAR_FILE=$(ls shanoir-uploader-*-jar-with-dependencies.jar 2>/dev/null | head -n 1)
```

Comment:
Hi @LaurentPV,
I am not a big fan of this ls/for calls to detect the jar-version.
One idea, why do we not always use a .jar, with the same name: shanoir-uploader-with-dependencies.jar?
The version will be in the execs produced by jlink and on Github.
Actually, as longer I think about it, I think we do too much here.
We can even fix the maven pom.xml to version 1.0.0 and never touch it.
As long as we know the release on Github with the packages produced,
what does it help us, that maven has the same version? as long as we tag
the code and can go back to it, what do we gain, when maven aligns with it?
With kind regards, Michael